src/routes/(site)/teams/team-section.svelte

```diff
@@ -37,18 +36,24 @@
       class:gamedev-animation={info.id === 'gamedev'}
     >
       {#if info.id === 'general'}
-        <DiamondPicture
-          src="https://cdn.discordapp.com/icons/710225099923521558/a_f72bf9caa196d84a44ff40cdfd3f8d9a.gif?size=1024"
-          reducedMotionSrc={info.logoSrc}
-          altSrc="General Picture"
-        />
+        <picture>
```

Comment:
The picture is not centered on the page.

Reply:
Fixed!! ty :capylove: